tests/client_test.py

```diff
@@ -328,6 +343,28 @@ def setUp(self):
                                         ]
                                     }
                                 ]
+                            },
+                            {
+                                "path": "/pet/{petId}/vaccine",
+                                "operations": [
+                                    {
+                                        "method": u"POST",
```

Comment:
Will do. Really, all of these should be unicode since that's what Response.json() returns. Going to cut a followup branch once we have a little more bandwidth.